src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -905,6 +905,17 @@ void QgsWmsCapabilities::parseLayer( QDomElement const & e, QgsWmsLayerProperty&
 
         parseStyle( e1, styleProperty );
 
+        for ( int i = 0; i < layerProperty.style.size(); ++i )
+        {
+          if ( layerProperty.style[i].name == styleProperty.name )
```

Comment:
I didn't know that QVector::at is const only and I see now why it would be faster.

I'll change it to make you happy, even though i doubt it changes anything since this instance is being changed in the next 3 lines of code. It's indeed a good reflex to have.